src/styles/blocks/card.scss

```diff
@@ -0,0 +1,80 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  box-sizing: border-box;
+  margin: 0 auto;
```

Comment:
7'th point in checklist, use mix instead

Reply:
This margin is just for beauty here and I can't to create parent block for block .card, so I don't know how to solve it for exeption just delete it.